caom2-artifact-resolvers/src/main/java/ca/nrc/cadc/caom2/artifact/resolvers/SubaruResolver.java

```diff
@@ -40,27 +41,18 @@
  *
  * @author jeevesh
  */
-public class SmokaResolver implements StorageResolver {
+public class SubaruResolver implements StorageResolver {
+    private static final Logger log = Logger.getLogger(SubaruResolver.class);
     private static final String SCHEME = "subaru";
-    private static final String FILE_URI = "file";
+    private static final String RAW_DATA_URI = "raw";
     private static final String PREVIEW_URI = "preview";
-    private static final String FILE_URL_QUERY = "object=&resolver=SIMBAD&coordsys=Equatorial&equinox=J2000&fieldofview=auto"
-        + "&RadOrRec=radius&longitudeC=&latitudeC=&radius=10.0&longitudeF=&latitudeF=&longitudeT=&latitudeT"
-        + "=&date_obs=&exptime=&observer=&prop_id=&frameid=&dataset=&asciitable=Table"
-        + "&frameorshot=Frame&action=Search&instruments=SUP&instruments=HSC&multiselect_0=SUP&multiselect_0=HSC"
-        + "&multiselect_0=SUP&multiselect_0=HSC&obs_mod=IMAG&obs_mod=SPEC&obs_mod=IPOL&multiselect_1=IMAG&multiselect_1=SPEC"
-        + "&multiselect_1=IPOL&multiselect_1=IMAG&multiselect_1=SPEC&multiselect_1=IPOL&data_typ=OBJECT&multiselect_2=OBJECT"
-        + "&multiselect_2=OBJECT&bandwidth_type=FILTER&band=&dispcol=FRAMEID&dispcol=DATE_OBS&dispcol=FITS_SIZE&dispcol=OBS_MODE"
-        + "&dispcol=DATA_TYPE&dispcol=OBJECT&dispcol=FILTER&dispcol=WVLEN&dispcol=DISPERSER&dispcol=RA2000&dispcol=DEC2000"
-        + "&dispcol=UT_START&dispcol=EXPTIME&dispcol=OBSERVER&dispcol=EXP_ID&orderby=FRAMEID&diff=100&output_equinox=J2000&from=0"
-        + "&exp_id="; //&exp_id=SUPE01318470 or similar for last entry here.
+
+    private static final String BASE_DATA_URL = "http://www.canfar.net/maq/subaru?frameinfo=";
```

Comment:
The whole MAQ project is a CADC activity. Why is this page hosted on www.canfar.net?

Reply:
It was easier to get up and running.  Is it misleading?